demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -257,6 +259,20 @@ def verify_no_default_sections_left(self) -> bool:
 
         return is_valid
 
+    def verify_readme_is_not_too_short(self):
+        is_valid = True
+        with open(self.file_path) as f:
```

Comment:
Instead of reading the file every time in each function, let's move this operation to the init function.